This check is broken in cross, see https://gitlab.gnome.org/GNOME/vala/issues/775

We are sure that we have cgraph, so just hack around this for now.

diff --git configure.ac configure.ac
index 16ebd1f..df789b9 100644
--- configure.ac
+++ configure.ac
@@ -164,25 +164,8 @@ if test x$enable_valadoc = xyes; then
 	cgraph_tmp_CFLAGS="$CFLAGS"
 	LIBADD="$LIBADD $LIBGVC_LIBS"
 	CFLAGS="$CFLAGS $LIBGVC_CFLAGS"
-	AC_RUN_IFELSE(
-		[AC_LANG_SOURCE([
-			#include <gvc.h>
-			int main(void) {
-				#ifdef WITH_CGRAPH
-					return 0;
-				#else
-					return -1;
-				#endif
-			}
-		])], [
-			AC_MSG_RESULT([yes])
-			VALAFLAGS="$VALAFLAGS -D WITH_CGRAPH"
-			have_cgraph=yes
-		], [
-			AC_MSG_RESULT([no])
-			have_cgraph=no
-		]
-	)
+	VALAFLAGS="$VALAFLAGS -D WITH_CGRAPH"
+	have_cgraph=yes
 	LIBADD="$cgraph_tmp_LIBADD"
 	CFLAGS="$cgraph_tmp_CFLAGS"
 fi
